airflow/www/templates/airflow/dags.html

```diff
@@ -75,7 +75,7 @@ <h2>DAGs</h2>
         <thead>
           <tr>
             <th width="12">
-              <span id="pause_header" class="material-icons text-muted" title="Use this toggle to pause a DAG. The scheduler won't schedule new tasks instances for a paused DAG. Tasks already running at pause time won't be affected.">info</span>
+              <span class="material-icons text-muted js-tooltip" title="Use this toggle to pause a DAG. The scheduler won't schedule new tasks instances for a paused DAG. Tasks already running at pause time won't be affected.">info</span>
```

Comment:
This change is unrelated to this PR, but fixes a scripting interaction I discovered during my manual testing.